### PR TITLE
feat(webapp): refactor notification page

### PR DESCRIPTION
**Pull request by maeckes-infinitylabs**
## 🍰 Pullrequest
Refactors the notification page to add better visibility for mobile devices. 
It now uses a mixture of CSS Grid and Flex to have a responsive table for desktop views.
For mobile devices it breaks the table down in notification-row items for a better overview.

### Issues

- feature #6016 

### Todo

- [x] Update vue component test

### CODE REVIEW

File: webapp/components/NotificationsTable/NotificationsTable.vue [hunk 1]
```diff
@@ -1,62 +1,106 @@
...
+              </ds-flex-item>
+              <ds-flex-item>
+                <div>
+                  <base-card :wide-content="true">
+                    <ds-space margin-bottom="base">
+                      <client-only>
+                        <user-teaser
+                          :user="n.from.author"
+                          :date-time="n.from.createdAt"
+                          :class="{ 'notification-status': n.read }"
```
Comment (Elweyn) [inline webapp/components/NotificationsTable/NotificationsTable.vue:44]:
```suggestion
                          :user="notification.from.author"
                          :date-time="notification.from.createdAt"
                          :class="{ 'notification-status': notification.read }"
```

File: webapp/components/NotificationsTable/NotificationsTable.vue [hunk 2]
```diff
@@ -1,62 +1,106 @@
...
+                        />
+                      </client-only>
+                    </ds-space>
+                    <ds-text :class="{ 'notification-status': n.read, reason: true }">
+                      {{ $t(`notifications.reason.${n.reason}`) }}
```
Comment (Elweyn) [inline webapp/components/NotificationsTable/NotificationsTable.vue:49]:
```suggestion
                    <ds-text :class="{ 'notification-status': notification.read, reason: true }">
                      {{ $t(`notifications.reason.${notification.reason}`) }}
```

**Review by Elweyn — CHANGES_REQUESTED** [review]
It is more a question to everyone do we use ``f`` or ``n`` in a loop or do we use the full descriptiv name, I prefer the second.

**Comment by maeckes-infinitylabs** [comment]
@Elweyn I fully agree with you to use full descriptive names for variables and added your suggestions

**Review by Elweyn — APPROVED** [review]
Great, I have no more objections.

**Review by Tirokk — APPROVED** [review]
Hey @maeckes-infinitylabs 

that looks really cool and breaks nicely into the mobile style.
Great 😍

I approve 🚀🚀💫💫

**Pull request merged**